R/grids.R

```diff
@@ -252,7 +253,9 @@ make_random_grid <- function(...,
 
 new_param_grid <- function(x = new_data_frame()) {
   if (!is.data.frame(x)) {
-    rlang::abort("`x` must be a data frame to construct a new grid from.")
+    cli::cli_abort(
+      "{.arg x} must be a data frame to construct a new grid from."
+    )
```

Comment:
Can you thread the call through and give this message a test?

Reply:
I can thread through the call. but this code as far as I can seen isn't possible to reach naturally. It is only applied to the output of functions like `make_max_entropy_grid()` and `make_latin_hypercube_grid()` which returns